pom.xml

```diff
@@ -221,8 +220,8 @@
                             <configuration>
                                 <attachSources>true</attachSources>
                                 <excludeArtifactIds>tools-jar,servlet-api,jakarta.faces</excludeArtifactIds>
-                                <excludes>javax/help/**,${extra.excludes}</excludes>
-                                <includes>javax/**, resources/**</includes>
+                                <excludes>jakarta/help/**,${extra.excludes}</excludes>
```

Comment:
I don't believe there **is** a jakarta.help.  Definitely there should **not** be.

Reply:
Thanks, @bshannon.  I didn't even know where the `javax/help` was coming from, so I just followed suit and assumed there was something to exclude...  Do you know the origin of the `javax/help?

Reply:
There's an official Java Help spec.

We were using it in the GlassFish admin console, but I think we purged it because no
one was maintaining it.

Reply:
Thanks, Bill.